src/web/screens/home.tsx

```diff
@@ -1,37 +1,47 @@
 import log from 'loglevel';
-import { Suspense, useContext } from 'react';
-import { Await, defer, useAsyncError, useLoaderData } from 'react-router-dom';
+import { useContext, useEffect, useState } from 'react';
 
+import { Loading } from '../components/Core/Loading';
 import DocumentationCard from '../components/Home/DocumentationCard';
 import SharingPermissionCard from '../components/Home/SharingPermissionCard';
 import { CurrentUserContext } from '../contexts/CurrentUserProvider';
-import { GetSharingParticipants, ParticipantResponse } from '../services/participant';
+import { GetSharingParticipants } from '../services/participant';
 import { PortalRoute } from './routeUtils';
 
 import './home.scss';
 
-function ErrorElement() {
-  const error = useAsyncError();
-  log.error(error);
-  return <SharingPermissionCard error={error} />;
-}
-
 function Home() {
   const { LoggedInUser } = useContext(CurrentUserContext);
-  const { sharingPermissions } = useLoaderData() as {
-    sharingPermissions: ParticipantResponse[];
-  };
+  const [loading, setIsLoading] = useState<boolean>(false);
+  const [sharingPermissionsCount, setSharingPermissionsCount] = useState<number>(0);
+  const [hasError, setHasError] = useState<boolean>(false);
+  useEffect(() => {
+    const getSharingParticipantsCount = async () => {
+      setIsLoading(true);
```

Comment:
I can see a flicker of "0" before loading is set to true - wouldn't it make more sense to start loading as true?